docs/docs/modeling.md

```diff
@@ -112,6 +113,40 @@ classDiagram
 
 ## Tenant - テナント
 
+### Tenant ID (Identifier)
+
+The Tenant ID is the UUID.
+
+### Name
+
+The Tenant has a name.
+
+Constraints:
+
+- at least one character no more than 15 characters.
+- unique (no duplicates).
+- does not have to be the legal name; an arbitrary name can be used instead.
+- can be changed anytime.
+
+### Email Address (Value Object)
+
+The Tenant has an email address.
+
+Constraints:
+
+- unique.
+- should be recognized consistently, irrespective of capitalization.
+- cannot be changed after sending invitation mail.
+
+### extraAssignedCount
+
+The default number of The extraAssignedCount is 0. If the number of Categories is larger than the number of Tenants, extraAssignedCount is used to ensure that an equal number of tasks(Category) are assigned to each Tenant.
```

Comment:
Thank you, Satomi:) It's a minor quibble, but I think you should fix it to either singular or plural since they are mixed up!